src/commands/UploadBrowserCommand.ts

```diff
@@ -22,6 +22,14 @@ export default async function uploadBrowser (argv: string[], opts: Record<string
     process.exitCode = 1
     if (e.name === 'UNKNOWN_VALUE') {
       logger.error(`Invalid argument provided. ${e.message}`)
+
+      // Check if the user provided an argument that allows a wildcard ('*') and
+      // if so, warn them about wrapping the value in quotes
+      const wildcardArgument = argv.find(arg => arg === '--bundle-url' || arg === '--base-url')
+
+      if (wildcardArgument) {
+        logger.info(`Values that contain a wildcard must be wrapped in quotes, for example ${wildcardArgument} "*"`)
```

Comment:
```suggestion
        logger.info(`Values that contain a wildcard must be wrapped in quotes to prevent shell expansion, for example ${wildcardArgument} "*"`)
```